.github/workflows/ci.yml

```
@@ -30,6 +30,12 @@ jobs:
           - windows-latest
         arch:
           - x64
+        exclude:
+          - github-runner: macos-latest # Apple Silicon
+            julia-version: '1.7'
```

```suggestion
            version: '1.7'
```
Does this need to match the `version` above (same with below)?